phoenicis-javafx/src/main/java/org/phoenicis/javafx/controller/ControllerConfiguration.java

```diff
@@ -143,4 +143,8 @@ public SettingsController settingsController() {
                 settingsConfiguration.settingsManager()
         );
     }
+
+    public String getWineEnginesPath() {
+        return this.wineEnginesPath;
```

Comment:
I'm not sure I understand how you would like to have it. Can you change it in another PR to have like you want to?